### PR TITLE
Add and tag features for the Publishing API

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/aW59k3Tf/170-activate-continuous-deployment-for-the-publishing-api

Please see the commits for more details.

### CODE REVIEW

File: features/publishing_tools.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@app-publishing-api
```
Comment (ChrisBAshton) [inline features/publishing_tools.feature:1]: What does this tagging actually achieve? Do we ever just run a subset of smokey tests by its tags, or does it produce more useful error output, or is this purely for categorisation / local development?

Comment (benthorner) [reply features/publishing_tools.feature:1]: Indeed we do: https://github.com/alphagov/govuk-puppet/pull/10387

Comment (ChrisBAshton) [reply features/publishing_tools.feature:1]: Excellent :) Will we have any corresponding docs on https://docs.publishing.service.gov.uk/?

Comment (benthorner) [reply features/publishing_tools.feature:1]: Yeah, that's on my list to do. I'd like to get this in if possible, and address the documentation separately.

I've made a start on that [here](https://github.com/alphagov/smokey/pull/683), if you'd like to review that 😃.

**Pull request merged**